tensornetwork/network_components.py

```diff
@@ -1201,6 +1279,17 @@ def __or__(self, other: "Edge") -> Tuple["Edge", "Edge"]:
       raise ValueError('Cannot break two unconnected edges')
     return self.disconnect()
 
+  def to_serial_dict(self) -> Dict:
+    """Return a serializable dict representing the edge.
+    
+    Returns: A dict object.
+    """
+    edge_dict = {
+        'name': self.name,
+        'axes': self._axes,
+    }
+    return edge_dict
+
```

Comment:
Do we not want to store any node information with the edge? I guess the cyclic nature of it makes it difficult...

Reply:
Yeah I think we want to normalize. Nodes are stored, edges are stored with ids pointing to the nodes they are linked to.